Public/Src/FrontEnd/UnitTests/Libs/Prelude.AmbientHacks.ts

```diff
@@ -59,7 +59,7 @@ namespace _PreludeAmbientHack_Transformer {
     export declare function writeData(destinationFile: Path, content: any, tags?: string[], description?: string): DerivedFile;
     export declare function writeAllLines(destinationFile: Path, contents: any[], tags?: string[], description?: string): DerivedFile;
     export declare function writeAllText(destinationFile: Path, content: string, tags?: string[], description?: string): DerivedFile;
-        export declare function sealDirectory(root: Directory, files: File[], tags?: string[], description?: string): FullStaticContentDirectory;
+    export declare function sealDirectory(root: Directory, files: File[], tags?: string[], description?: string, scrub?: boolean): FullStaticContentDirectory;

```

Comment:
>root [](start = 42, length = 4)

After chatting, we should probably clean this api up and make it take an object literal.
I'll file a workitem on myself to do this in the future.
Let me know if you are going to add more.

Reply:
I don't see it will happen in the near future.